schema.sql

```diff
@@ -7,4 +7,9 @@ CREATE TABLE animals (
 	escape_attemps INT,
 	neutered BOOLEAN,
 	weight_kg DECIMAL
-);
\ No newline at end of file
+);
+
+/* Add a column species of type string to your animals table. */
+
+ALTER TABLE animals
+ADD COLUMN species text;
```

Comment:
* Kindly use the right data type of string for the species column [Check requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Add%20a%20column%20species%20of%20type%20string%20to%20your%20animals%20table.%20Modify%20your%20schema.sql%20file.) read more [on data types for strings ](https://www.tutlane.com/tutorial/sql-server/string-data-types-in-sql)